.gitignore

```diff
@@ -1,4 +1,5 @@
 /vendor/
+composer.lock
```

Comment:
it is a good practice to commit the lock file as well.

Reply:
yeah but we have to locally update composer.lock file every time composer.json file changes

Reply:
>  we have to locally update composer.lock file every time we need an updated version

how is the update done? Won't the `composer` command update the lock file as well?